nock/translation/analyze-plain-text-sentiment/ibm-cloud-natural-language-understanding-eu-de/AnalyzePlainTextSentiment/recording.json

```diff
@@ -0,0 +1,348 @@
+[
+  {
+    "scope": "https://api.eu-de.natural-language-understanding.watson.cloud.ibm.com:443",
+    "method": "POST",
+    "path": "/instances/b9f4f39c-c3e0-467e-b00e-f1756259557c/v1/analyze?version=2021-08-01",
+    "body": {
+      "text": "Enjoy your stay!",
+      "language": "en",
+      "features": {
+        "sentiment": {
+          "document": true
+        }
+      }
+    },
+    "status": 200,
+    "response": {
+      "usage": {
+        "text_units": 1,
+        "text_characters": 16,
+        "features": 1
+      },
+      "sentiment": {
+        "document": {
+          "score": 0.956482,
+          "label": "positive"
+        }
+      },
+      "language": "en"
+    },
+    "rawHeaders": [
+      "Server",
+      "watson-gateway",
+      "Content-Length",
+      "203",
+      "Content-Type",
+      "application/json; charset=utf-8",
+      "Cache-Control",
+      "no-cache, no-store",
+      "x-dp-watson-tran-id",
+      "41d942fa-2cf5-460b-bd6f-d38affb1df23",
+      "Content-Security-Policy",
+      "default-src 'none'",
+      "Pragma",
+      "no-cache",
+      "x-content-type-options",
+      "nosniff",
+      "x-frame-options",
+      "DENY",
+      "x-xss-protection",
+      "1; mode=block",
+      "strict-transport-security",
+      "max-age=31536000; includeSubDomains;",
+      "x-dp-watson-tran-id",
+      "41d942fa-2cf5-460b-bd6f-d38affb1df23",
+      "x-request-id",
+      "41d942fa-2cf5-460b-bd6f-d38affb1df23",
+      "x-global-transaction-id",
+      "41d942fa-2cf5-460b-bd6f-d38affb1df23",
+      "X-EdgeConnect-MidMile-RTT",
+      "0",
+      "X-EdgeConnect-Origin-MEX-Latency",
+      "146",
+      "Date",
+      "Tue, 05 Oct 2021 09:55:29 GMT",
+      "Connection",
+      "close"
+    ],
+    "reqheaders": {
+      "accept": [
+        "*/*"
+      ],
+      "authorization": [
+        "Basic YXBpa2V5OnJlZGFjdGVk"
```

Comment:
@janhalama is this OK?

Reply:
+ it is repeated in other situations

Reply:
I updated testing-lib and was forced to update also one-sdk. Both in beta versions.